Release v3.8.5 (#169)
DeterminateSystems/nix-src: doc/manual/source/release-notes-determinate/changes.md

<!-- Determinate Nix version 3.8.4 -->

* Revert "Use WAL mode for SQLite cache databases" by @grahamc in [DeterminateSystems/nix-src#155](https://github.com/DeterminateSystems/nix-src/pull/155)

<!-- Determinate Nix version 3.8.5 -->

> * Apply upstream formatting changes by @edolstra in [DeterminateSystems/nix-src#159](https://github.com/DeterminateSystems/nix-src/pull/159)
> 
> * Avoid isValidPath(), use queryPathInfo() instead by @edolstra in [DeterminateSystems/nix-src#157](https://github.com/DeterminateSystems/nix-src/pull/157)
> 

Reviewer: ```suggestion
```

> * Imply --offline during tab completion by @edolstra in [DeterminateSystems/nix-src#161](https://github.com/DeterminateSystems/nix-src/pull/161)

Reviewer: ```suggestion
* Tab completing arguments to Nix avoids network access. [DeterminateSystems/nix-src#161](https://github.com/DeterminateSystems/nix-src/pull/161)
```

> * SQLite: fsync db.sqlite-shm before opening the database by @edolstra in [DeterminateSystems/nix-src#158](https://github.com/DeterminateSystems/nix-src/pull/158)

Reviewer: ```suggestion
* Nix on ZFS no longer stalls for multiple seconds at a time [DeterminateSystems/nix-src#158](https://github.com/DeterminateSystems/nix-src/pull/158)
```

> * Make GitFileSystemObjectSink multi-threaded by @edolstra in [DeterminateSystems/nix-src#149](https://github.com/DeterminateSystems/nix-src/pull/149)

Reviewer: ```suggestion
* Importing Nixpkgs and other tarballs to the cache 2-4x faster [DeterminateSystems/nix-src#149](https://github.com/DeterminateSystems/nix-src/pull/149)
```

> 
> * Sync with upstream 2.30.2 by @edolstra in [DeterminateSystems/nix-src#160](https://github.com/DeterminateSystems/nix-src/pull/160)

Reviewer: ```suggestion
```

> * Add paths to the store asynchronously by @edolstra in [DeterminateSystems/nix-src#162](https://github.com/DeterminateSystems/nix-src/pull/162)

Reviewer: ```suggestion
* Adding paths to the store is significantly faster [DeterminateSystems/nix-src#162](https://github.com/DeterminateSystems/nix-src/pull/162)
```
